modules/olm-arch-os-support.adoc

```diff
@@ -30,8 +30,6 @@ The following strings are supported in Operator Lifecycle Manager (OLM) on {prod
 |Linux
 |`linux`
 
-|z/OS
-|`zos`
 |===
 
 [NOTE]
```

Comment:
Is this note still needed?

Reply:
It still seemed relevant, particularly for the "other Kubernetes-based distributions" part.